app/helpers/custom_form_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CustomFormHelper
+  def submit_button(i18n_key, opts = {}, &block)
```

Comment:
Let's maintain if possible the same interface we had, because it will be needed when we get back to work on multiple convictions.

Basically, just replicating the `continue_button` (and `cancel_check_link`) methods we have here:
lib/govuk_components/form_builder.rb

But adapting them to use the new gem.